nvme_gw_cli.py

```diff
@@ -275,17 +275,26 @@ def delete_host(self, args):
 
     @cli.cmd([
         argument("-n", "--subnqn", help="Subsystem NQN", required=True),
-        argument("-a", "--traddr", help="NVMe host IP", required=True),
-        argument("-s", "--trsvcid", help="Port number", required=True),
+        argument("-g", "--gateway", help="Gateway name", default=""),
         argument("-t", "--trtype", help="Transport type", default="TCP"),
         argument("-f", "--adrfam", help="Address family", default="ipv4"),
+        argument("-a", "--traddr", help="NVMe host IP", default=""),
+        argument("-s", "--trsvcid", help="Port number", required=True),
     ])
     def create_listener(self, args):
         """Adds a listener at a particular TCP/IP address for a given subsystem."""
 
+        if args.gateway and not args.traddr:
```

Comment:
Why not let the RPC fail with the error?

Reply:
I thought it would be more clear for users but I don't mind. I will change.